makefiles/mcuboot.mk

```diff
@@ -42,12 +42,12 @@ $(MCUBOOT_BIN):
 .PHONY: mcuboot-flash-bootloader mcuboot-flash
 
 mcuboot-flash-bootloader: FLASHFILE = $(MCUBOOT_BIN)
-mcuboot-flash-bootloader: export FLASH_ADDR = 0x0
```

Comment:
This is unrelated to the PR, same for all the occurrences.

Reply:
removed from this PR